### PR TITLE
feat: replace theme footer

### PR DESCRIPTION
**Pull request by tabathadelane**
Footer styling now behaves similar to the Header and stretches to fill the page width while restricting it's contents to the site-max-width.

Screen Shot 2022-02-02 at 9 29 28 AM

Closes https://github.com/newrelic/developer-website/issues/2020

### CODE REVIEW

File: src/@newrelic/gatsby-theme-newrelic/components/GlobalFooter.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+              size={Button.SIZE.SMALL}
+              instrumentation={{ component: 'GlobalFooter' }}
+            />
+          )}
+        </div>
+      </div>
+
+      <div
+        css={css`
+          background-color: rgba(0, 0, 0, 0.05);
```
Comment (clarkmcadoo) [inline src/@newrelic/gatsby-theme-newrelic/components/GlobalFooter.js:121]: do we have variables for these anywhere? if not just disregard

**Comment by clarkmcadoo** [comment]
Just to double-check, this is for the footer right? Just looking at the title 🙈

**Review by clarkmcadoo — DISMISSED** [review]

File: src/@newrelic/gatsby-theme-newrelic/components/GlobalFooter.js (hunk 1 above)
Comment (tabathadelane) [reply src/@newrelic/gatsby-theme-newrelic/components/GlobalFooter.js:120]: I think this is just a transparent black overlay for the bottom half of the footer and not used in multiple places.

**Comment by tabathadelane** [comment]
> Just to double-check, this is for the footer right? Just looking at the title 🙈

yup. idk what i was doing...

**Review by clarkmcadoo — APPROVED** [review]

**Pull request merged**